src/AdvancedVI.jl

```diff
@@ -1,270 +1,157 @@
-module AdvancedVI
-
-using Random: Random
-
-using Distributions, DistributionsAD, Bijectors
-using DocStringExtensions
-
-using ProgressMeter, LinearAlgebra
 
-using ForwardDiff
-using Tracker
+module AdvancedVI
 
-const PROGRESS = Ref(true)
-function turnprogress(switch::Bool)
-    @info("[AdvancedVI]: global PROGRESS is set as $switch")
-    PROGRESS[] = switch
-end
+using SimpleUnPack: @unpack, @pack!
+using Accessors
 
-const DEBUG = Bool(parse(Int, get(ENV, "DEBUG_ADVANCEDVI", "0")))
+using Random: AbstractRNG, default_rng
+using Distributions
+import Distributions:
+    logpdf, _logpdf, rand, rand!, _rand!,
+    ContinuousMultivariateDistribution
```

Comment:
Avoid usage of `import`, and instead use the qualified names of the functions when you define new dispatches for them. Otherwise it's very difficult for the reader to know where the original method is defined.

Reply:
done